resources/Materials/TestSuite/Utilities/closure_color_scene.xml

```diff
@@ -1,22 +1,21 @@
 <World>
   <Camera eye="0, 0, 4"  dir="0,0,-1"  fov="70" />
 
-   <!-- Background clear. Uncomment this use flat color
-   %background_color% : is the background color
-   -->
-   <!--
+   <!-- Background environment map. -->
    <ShaderGroup>
-      color Cin %background_color%;
-      shader constant_color_background layer1;
+      %environment_shader_parameter_overrides%
+      shader envmap layer1;
    </ShaderGroup>
    <Background resolution="1024" />
+
+   <!-- Background quad. 
+   %background_color% : is the background color
    -->
-   <!-- Background environment map. Comment this out to use flat color  -->
    <ShaderGroup>
-      %environment_shader_parameter_overrides%
-      shader envmap layer1;
+      color Cin %background_color%;
```

Comment:
Bit of a hack to cover up the environment background with a quad.

Reply:
Yeah this will probably affect reflections (indirect light) as well since secondary rays will hit this quad. So might give different shading result on the sphere, not just in the background.

But I know this comes from the lack of control in testrender, where you cannot set different environments for different ray types, to get a uniform background color seen from the camera. I've tried to work around it myself with no success. The proper fix would be to implement the missing support for the `raytype()` OSL function in testrender. But of course this is out of scope for this review.

Reply:
Yes, A few of the results are little bit brighter overall but most are hopefully not noticeable as I placed the plane fairly far away.